modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/plugins/experience/components/ExperienceSelector.js

```diff
@@ -343,6 +368,9 @@ const ExperienceSelector = ({
 								}
 								experiences={experiences}
 								onDeleteExperience={deleteExperience}
+								onDuplicateExperience={
```

Comment:
Maybe we can apply a little bit of renaming here. They are not changes from this feature, but I think we can organize a little the naming for the actions because we have `handleXXX`, `XXXExperience` and `handleXXXClick` and `XXXPriority` 🤦‍♀️  I mean:

```js
onDeleteExperience={deleteExperience}
onDuplicateExperience={handleExperienceDuplication}
onEditExperience={handleEditExperienceClick}
onPriorityDecrease={decreasePriority}
onPriorityIncrease={increasePriority}
```

What do you think? :)

Reply:
That's true, but d you think It's a good idea to do it in this PR? Maybe we should create a dedicated one, shouldn't we?

Reply:
Sure! https://issues.liferay.com/browse/LPS-126155 😉

Reply:
wow, thanks!!